R/module_teal.R

```diff
@@ -98,14 +95,37 @@ ui_teal <- function(id,
     )
   )
 
+  data_elem <- ui_data(ns("data"), data = data, title = title, header = header, footer = footer)
+  tabs_elem <- ui_teal_module(id = ns("root_module"), modules = modules)
+
   fluidPage(
     title = title,
     theme = get_teal_bs_theme(),
     include_teal_css_js(),
     tags$header(header),
     tags$hr(class = "my-2"),
     shiny_busy_message_panel,
-    splash_ui,
+    tabs_elem,
+    tags$div(
+      id = "teal-util-icons",
+      style = "margin-left: auto;",
+      data_elem,
+      ui_bookmark_panel(ns("bookmark_manager"), modules),
+      tags$button(
+        class = "btn action-button filter_hamburger", # see sidebar.css for style filter_hamburger
+        href = "javascript:void(0)",
+        onclick = "toggleFilterPanel();", # see sidebar.js
+        title = "Toggle filter panel",
+        icon("fas fa-bars")
+      ),
```

Comment:
Can we please use `actionButton`? This automatically adds a class `"btn-default"` which might be required for auto-styling with `bslib` or anything like that. See: https://github.com/insightsengineering/teal/issues/1242
If you spot something similar - please also fix that